infra/Pos/Network/Windows/DnsDomains.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE CPP                      #-}
+{-# LANGUAGE ForeignFunctionInterface #-}
+
+#if defined(mingw32_HOST_OS)
+#define WINDOWS
+#endif
+
+module Pos.Network.Windows.DnsDomains (
+#ifdef WINDOWS
+    getWindowsDefaultDnsServer
+#endif
+    ) where
+
+#ifdef WINDOWS
+import           Universum
+
+import           Foreign.C            (CString, peekCString)
+
+foreign import ccall "getWindowsDefDnsServer" getWindowsDefDnsServer :: IO CString
+
+getWindowsDefaultDnsServer :: IO String
```

Comment:
Could you add some documentation here, with an example output of what the resulting string looks like?